styles/components/button.scss

```diff
@@ -1,56 +1,40 @@
-// @mixin btn-variant($local-bg, $local-color: $white) {
-//   color: $local-color;
-//   background: $local-bg;
-
-//   &:hover {
-//     color: $local-color;
-//     background: darken($local-bg, 5%);
-//   }
-
-//   &.-disabled,
-//   &:disabled,
-//   &.-disabled:hover,
-//   &:disabled:hover {
-//     background: lighten($local-bg, 20%);
-//   }
-// }
-
-// @mixin btn-variant-bordered($local-bg, $local-color: $white) {
-//   color: $local-bg;
-//   background: transparent;
-//   border-color: $local-bg;
-
-//   &:hover {
-//     color: darken($local-bg, 5%);
-//     background: transparent;
-//     border-color: darken($local-bg, 5%);
-//   }
-
-//   &.-disabled,
-//   &:disabled,
-//   &.-disabled:hover,
-//   &:disabled:hover {
-//     color: lighten($local-bg, 20%);
-//     border-color: lighten($local-bg, 20%);
-//   }
-// }
+@mixin btn-variant($local-bg, $local-color) {
+  color: $local-color;
+  background: $local-bg;
+  border: 3px solid $local-bg;
+
+  &:hover,
+  &:focus {
+    color: $local-color;
+    background: darken($local-bg, 5%);
+  }
+}
+
+@mixin btn-variant-inverted($local-color, $local-bg: $brand-white) {
+  color: $local-color;
+  background: $local-bg;
+  border: 3px solid $local-color;
+
+  &:hover,
+  &:focus {
+    color: $local-bg;
+    background: $local-color;
+    border: 3px solid $local-color;
```

Comment:
If you only have to change the border, why bother putting the entire properties? Use `border-color` if you're only changing the color.